publications-2022.bib

```diff
@@ -2563,6 +2563,13 @@ @Book{2022:wick:numerical
   copyright = {CC BY-NC-ND 3.0 DE}
 }
 
+@Misc{2022:wick:space-time,
+  author  = {Thomas Wick},
+  title   = {Space-time methods},
+  year    = 2022,
+  url     = {https://thomaswick.org/links/Wi22_st_book_preprint_Dec_6_2022.pdf}
```

Comment:
For some reason, the link only works with `www`. While there, I suggest to be explicit about it being a preprint (as the link suggests)
```suggestion
  note    = {Preprint},
  url     = {https://www.thomaswick.org/links/Wi22_st_book_preprint_Dec_6_2022.pdf}
```